tests/end_to_end/features/constants.py

```diff
@@ -0,0 +1,11 @@
+import os
+
+SUBSCRIPTION_ID = os.environ.get("AZURE_SUBSCRIPTION_ID")
```

Comment:
Nit picking, but I like to have the Constant be the same as the env var, just personal pref.

Reply:
Agree updated here d604bbc98de94c95f7894bb601d02de8222d4f63